apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts

```diff
@@ -303,9 +303,9 @@ class FanClubMembershipController extends BaseController {
     fanClubId,
     memberId,
   }: FanClubMembershipRequestParams): FanClubMembershipRequestResult {
-    let isBulkAction = false;
+    let isBulkAction = true;
     if (!Array.isArray(fanClubId)) {
-      isBulkAction = true;
+      isBulkAction = false;
```

Comment:
why we need bulkAction here?

Reply:
I just write dynamic create action, if provided data is single then will be created only one row, but if the client-side send us a list of rows in the same route then we can execute the action for bulk insertion.

Just have one route for single and bulk action.

Reply:
So what's the use-case of using bulk action?

Reply:
In UI we don't have bulk actions for now, but maybe someday we need functionality to allow members to select more than one fan club and request to join.